core/src/main/java/net/sourceforge/jnlp/runtime/JNLPClassLoader.java

```diff
@@ -524,16 +522,16 @@ public static JNLPClassLoader getInstance(JNLPFile file, UpdatePolicy policy, St
     /**
      * Returns a JNLP classloader for the JNLP file at the specified location.
      *
-     * @param location the file's location
-     * @param uniqueKey key to manage applets/applications in shared vm
-     * @param version the file's version
-     * @param settings settings of parser
-     * @param policy the update policy to use when downloading resources
-     * @param mainName Overrides the main class name of the application
+     * @param location       the file's location
+     * @param uniqueKey      key to manage applets/applications in shared vm
+     * @param version        the file's version
+     * @param settings       settings of parser
+     * @param policy         the update policy to use when downloading resources
+     * @param mainName       Overrides the main class name of the application
```

Comment:
My ide is auto-formatting differently. The auto-formatter config file should be pushed into repo.